### ISSUE TITLE
Support reading of larger CORSIKA files

### ISSUE THREAD

**Issue by 153957** [HiSPARC/sapphire#46]
Currently the CorsikaFile class in SAPPHiRE does not work well for files largers than memory.
This is because it tries to read the entire file into memory..

Now that we have large showers (E > 10**17 eV) we need to be able to read (and convert) these larges files to h5.

This problem is mentioned in #21.

I will see if `file.seek(offset).read(size)` can be used to read the file with reasonable speed.

**Comment by 153957** [comment]
Works but is a bit slower for a file of 100 MB (56s -> 1m10s..)
Ill check for larger files.

**Comment by 153957** [comment]
I just converterd 

```
> time store_corsika_data DAT000000 corsika_new.h5
real	130m42.383s
user	60m53.542s
sys	1m37.739s

> time store_corsika_data DAT000000 corsika_old.h5  
real	98m7.960s
user	48m33.870s
sys	2m56.625s
```

**Comment by 153957** [comment]
Fixed in daab038a3014377f993798d43022cb0e6e621cf4
